### PR TITLE
Feature/us107 log in

### PR DESCRIPTION
**Pull request by duncanmcgowan**

### CODE REVIEW

**Comment by Pheonnexx** [comment]
This is not to be merged until we have acceptance tests for this.

File: tests/test_app.py [hunk 1]
```diff
@@ -1,37 +1,33 @@
+"""
```
Comment (russwillis) [inline tests/test_app.py]: Need to reinstate all unit tests temporarily removed as otherwise we'll lose them and our coverage rates.
Apart from that LGTM

**Comment by mattgirdler** [comment]
Apart from Russ's comment, looks good.

**Comment by Pheonnexx** [comment]
Acceptance tests are working now

File: service/server.py [hunk 2]
```diff
@@ -8,12 +7,12 @@
 import re
 import time
 
-from service import (address_utils, api_client, app, auditing, health_checker, login_api_client,
-                     login_manager, title_formatter, title_utils)
+from service import (address_utils, api_client, app, auditing, health_checker,
+                     title_formatter, title_utils)
 from service.forms import TitleSearchForm, SigninForm
 
+app.secret_key = 'F12Zr47j\3yX R~X@H!jmM]Lwf/,?KT'
```
Comment (russwillis) [inline service/server.py]: Why have we got a secret key in the server file? Shouldn't this be a environment variable?!

File: service/server.py [hunk 3]
```diff
@@ -23,31 +22,82 @@
...
 
-class User():
+#@app.before_request
+#def before_request():
+#    session['user_id'] = request.headers.get("iv-user", None)
+#    if session['user_id'] is None:
+#      return None
+#    p = re.compile("[%][{0-9}][{0-9}]")
+#    session['user_id'] = p.sub("", session['user_id'])
+#    return session['user_id']
```
Comment (russwillis) [inline service/server.py]: Do we still need these comments?

File: service/templates/confirm_selection.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <li>title number</li>
+        <li>who owns it</li>
+        <li>what they paid for it (properties only, if available)</li>
+        <li>any rights of way</li>
+      </ul>
+    </div>
+    <br/>
+    <dl class="definition-tabular font-xsmall">
+      <!---dt>Payment of</dt>
+      <dd><b>Summary of Title Register</b></dd-->
```
Comment (russwillis) [inline service/templates/confirm_selection.html]: commented out code. Is this needed?

File: service/templates/payment-processing.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  </div>
+
+</main>
+
+{#
+  <script type="text/javascript">
+  //var timeoutID = window.setTimeout(move, 1500);
+  //function move() {
+  //  window.location.href = '{{ url_for('payment_details', title_number=title_number, search_term=search_term, page=display_page_number, products=products_string) }}';
+  //}
```
Comment (russwillis) [inline service/templates/payment-processing.html]: Remove all this commented out code

File: service/templates/spinner-page.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    <!-- Powered by Worldpay logo-->
+    <a href="http://www.worldpay.com/" target="_blank" title="Payment Processing - Worldpay - Opens in new browser window"><img src="http://www.worldpay.com/images/poweredByWorldPay.gif" border="0" alt="Worldpay Payments Processing"></a>
+  </div>
+
+</main>
+
+<script type="text/javascript">
+{#
+//var timeoutID = window.setTimeout(move, 1500);
+//function move() {
```
Comment (russwillis) [inline service/templates/spinner-page.html]: delete this commented out code

**Comment by russwillis** [comment]
My comments have now been actioned. Just need to tidy up the commit history then all good to go.

**Pull request merged**